nodeup/pkg/model/secrets.go

```diff
@@ -128,6 +128,10 @@ func (b *SecretBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 	}
 
+	if err := b.BuildPrivateKeyTask(c, "master", "service-account.key"); err != nil {
```

Comment:
I was thinking about strategies if we ever did want to rename this from master -> service-account (or similar) without breaking things.  I think it's possible, but it's not entirely trivial.   I suggest we live with it until the OIDC signing thing evolves (e.g. rotation of these signing keys is coming at some stage, I think)

Reply:
I was just thinking of renaming this and adding a `kops rotate secret service-account` command. There would be three secrets: `service-account`, `service-account-previous`, and `service-account-next`.

We could do this at the VFS layer, having a read of the `service-account` keypair falling back to `master` and returning it with the `legacyFormat` bit set.